modules/twinklewarn.js

```diff
@@ -1114,47 +1125,74 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 
 		$.each(contents, function(itemKey, itemProperties) {
 			// Skip if the current template doesn't have a version for the current level
-			if (!!level && !itemProperties[value]) {
+			if (!!level && !itemProperties[val]) {
 				return;
 			}
-			var key = typeof itemKey === 'string' ? itemKey : itemProperties.value;
+			var key = typeof itemKey === 'string' ? itemKey : itemProperties.val;
```

Comment:
Neither `itemProperties.val` nor `itemProperties.value` make sense since AFAICT `itemProperties` doesn't have a property named `val` or `value`. But it seems the code never reaches here as `typeof itemKey === 'string'` is always true.

Reply:
This bit is for custom warning templates, which are brought in as something like:
```javascript
0: {
        "value": "Happy",
        "label": "Winter"
      },
1: ...
```
As such, they *do* have a different `typeof`, hence this line.  That being said, I apparently changed this overeagerly when previously refactoring above (`value` to `val`).  So... we were both wrong!  Fixed.